### ISSUE TITLE
[FEEDBACK] Codes in JSON file should be sorted by date descending

### ISSUE THREAD

**Issue by SalbugVR** [ChrisFeline/ToNSaveManager#1]
While it's nice that your codes are being auto saved to a json file, it's currently hard to parse through and find the latest save especially once the file starts to get large. Codes should be sorted by date descending so that the latest save code always appears at the top first.

**Comment by ChrisFeline** [comment]
Thank you for your feedback.

`Dictionary<TKey, TValue>` it's inherently unordered, and the purpose of the `data.json` is to store previous values, so if VRChat deletes log files you can still access your old saved codes. It's a database with formatted content, not meant to be user friendly or readable.
